repos/core-sdk.proj

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <PreferredHash>118dd862c853ff39694449ac303f5d8ef7d11b24</PreferredHash>
+    <PreferredHash>b487ff10aafb8b58eaa2b78a236b1e97999d7a22</PreferredHash>
   </PropertyGroup>
 
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
 
   <PropertyGroup>
-    <OfficialBuildId>20190416.21</OfficialBuildId>
+    <OfficialBuildId>fix me</OfficialBuildId>
```

Comment:
This seems like it needs an update.

Reply:
Thanks for the catch, added the actual build and version numbers.